### ISSUE TITLE
Dropdown Menu is not accessible for NVDA

### ISSUE THREAD

**Issue by petroholo** [foundation/foundation-sites#12390]
## What should happen?

User can expand dropdown when using NVDA

## What happens instead?

User can't expand dropdown when using NVDA

## Test Case and/or Steps to Reproduce (for bugs)

Try to expand items on this example with NVDA https://codepen.io/IamManchanda/pen/XRYWPO?editors=1000

**Comment by joeworkman** [comment]
I have no clue what NVDA is.
